spec/Task/Git/BranchNameSpec.php

```diff
@@ -107,4 +110,16 @@ function it_runs_with_detached_head_setting(RunContext $context, GrumPHP $grumPH
         $result = $this->run($context);
         $result->isPassed()->shouldBe(false);
     }
+
+    function it_runs_with_blacklisted_items(RunContext $context, GrumPHP $grumPHP, Repository $repository)
+    {
+        $grumPHP->getTaskConfiguration('git_branch_name')->willReturn([
+            'blacklist' => ['master'],
```

Comment:
Maybe test with multiple blacklisted items, to see if one match results in a failure?